gui/assets/public/css/pagination.css

```diff
@@ -58,8 +58,8 @@
   background-position: center;
 }
 .paginationjs .pagination-arrow-right {
-  background-image: url(/images/right_arrow.svg);
+  background-image: url('/assets/images/right_arrow.svg');
```

Comment:
Why has the directory structure changed for gui assets?

Reply:
This is done to simplify the `route()` function in `gui.go`:

![Screenshot from 2020-04-16 10-04-47](https://user-images.githubusercontent.com/6762864/79437244-d5c78c00-7fc9-11ea-84e4-3ac8fa5f3bc6.png)

Using a common root directory for all assets means this code only has to appear once, otherwise all of this code needs to be duplicated for each directory (3x for `images`, `js`, `css`). 

If we were to add further directories in future, that code would have to be duplicated again.

Further, I expect we will want to add extra code for this router in future (eg. `cache-control` HTTP headers), and that would also need to be duplicated 3 times.

Reply:
noted, thanks.